frontend/src/components/regions/CardDetails/utils.test.js

```diff
@@ -0,0 +1,25 @@
+import {trimReviewComments } from "./utils";
+
+test("A review with markdown should be cleaned", () => {

```

Comment:
This should explain what should happen when what has happened (I/O)

all of them, your test cases -> 

When this fails, Yes, would be easier to see where it is in the code, but how would one know exactly what they are testing for?